src/charts/heatmap/heatmap.directive.js

```diff
@@ -144,25 +154,36 @@ angular.module('patternfly.charts').directive('pfHeatmap', function ($compile, $
       showLegend: '=?',
       legendLabels: '=?',
       maxBlockSize: '@',
+      minBlockSize: '@',
       blockPadding: '@',
       thresholds: '=?',
       heatmapColorPattern: '=?',
-      clickAction: '=?'
+      clickAction: '=?',
+      rangeOnHover: '=?',
+      rangeHoverSize: '@',
+      rangeTooltips: '=?'
     },
     templateUrl: 'charts/heatmap/heatmap.html',
     controller: function ($scope) {
       var thresholdDefaults = [0.7, 0.8, 0.9];
       var heatmapColorPatternDefaults = ['#d4f0fa', '#F9D67A', '#EC7A08', '#CE0000'];
       var legendLabelDefaults = ['< 70%', '70-80%' ,'80-90%', '> 90%'];
+      var rangeTooltipDefaults = ['< 70%', '70-80%' ,'80-90%', '> 90%'];
       var heightDefault = 200;
 
       //Allow overriding of defaults
+      if ($scope.minBlockSize === undefined || isNaN($scope.minBlockSize)) {
+        $scope.minSize = 5;
```

Comment:
I read this in Google doc; “In angular patternfly we set the max size to be 50px but the application can set the max for a particular heat map to be from 5-50”. I assumed we don't want to make it smaller than that?

Do we want to allow it to be 2x2 only when range hover is enabled?

Regarding the check after determineBlockSize, we never had a test for min block size. I'll look at adding one.